src/features/keystrokes/KeystrokesHelper.tsx

```diff
@@ -0,0 +1,20 @@
+export const ONE_HOUR_IN_MILLISECONDS = 3600000;
+const ONE_BYTE_IN_BITS = 8;
+const ONE_KB_IN_BYTES = 1024;
+const FIVE_MB_IN_KB = 5 * 1024;
```

Comment:
SHOUTY convention is saved for actions, just use camelCase for this. Additionally, constants should be in `KeystrokesTypes.ts` instead